src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/GridView/ItemTemplateSelectorTestTemplateSelector.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Presentation.SamplePages;
+
+namespace Uno.UI.Samples.Content.UITests
+{
+	public class ItemTemplateSelectorTestTemplateSelector : DataTemplateSelector
+	{
+		public DataTemplate DefaultTemplate { get; set; }
+		public DataTemplate RedTemplate { get; set; }
+		public DataTemplate BlueTemplate { get; set; }
+		public DataTemplate GreenTemplate { get; set; }
+
+		protected override DataTemplate SelectTemplateCore(object item)
+		{
+			var colourString = (item as ItemTemplateSelectorTestPageViewModel.ListItem)?.ColourString;
+
+			bool? nullableBool = null;
+			var toBool = nullableBool ?? false;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'toBool'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5929295)